data/snippet/rewrite_urls.toml

```diff
@@ -27,7 +18,16 @@ class AttributeRewriter {
   }
 }
 
```

Comment:
Does this work? I am pretty sure it will break with the newlines here

Reply:
What do you mean by work? Like render in the UI? I'm able to render it in the UI, and this code has been tested too!